### PR TITLE
Adds identity definition to the jsonschema schema

### PR DESCRIPTION
**Pull request by jspaaks**
Note the merge target is not `main` but `1.2.0`.

**How to test**

```shell
git clone this repo
git checkout this branch
python3 -m venv env
source env/bin/activate
pip install --upgrade pip setuptools wheel
cd schema-sandbox
pip install -r requirements.txt
python3 schema_poc.py -d data.yml -s schema.json  # <- should be quiet
```

### CODE REVIEW

File: schema-sandbox/schema.json [hunk 1]
```diff
@@ -30,15 +30,22 @@
...
         "commit": {
             "type": "string",
             "description": "The commit SHA1 hash or revision number of the software version",
             "pattern": "^[a-f0-9]{7,40}$"
+        },
+        "identifiers": {
+            "items": {
+                "$ref": "#/definitions/identifier"
+            },
+            "minItems": 1,
```
Comment (sdruskat) [inline schema-sandbox/schema.json:47]: Shouldn't this also be unique items?

```suggestion
            "minItems": 1,
             "uniqueItems": true,
```

**Review by sdruskat — APPROVED** [review]
Thanks, this looks good to me, but I think identifiers could also be `uniqueItems`?
Let me know if this is okay for you and then this is good to be merged (feel free to do so).

File: schema-sandbox/schema.json (hunk 1 above)
Comment (jspaaks) [reply schema-sandbox/schema.json:47]: added

**Comment by jspaaks** [comment]
Thanks for reviewing!

**Pull request merged**